data/cybox/nozomi/nozomi_alerts_22022024.json

```diff
@@ -0,0 +1,563 @@
+                    "dst_ref": "1",
+                    "src_ref": "3",
+                    "dst_port": 445,
+                    "src_port": 34543,
+                    "protocols": [
+                        "smb",
+                        "tcp"
+                    ]
+                },
+                "3": {
+                    "type": "ipv4-addr",
+                    "value": "4.4.4.4",
+                    "resolves_to_refs": [
+                        "5"
+                    ],
+                    "x_nozomi_info_ref": "7"
+                },
+                "4": {
+                    "type": "mac-addr",
+                    "value": "00:04:xx:e0:xx:1c",
+                    "x_nozomi_info_ref": "6"
+                },
+                "5": {
+                    "type": "mac-addr",
+                    "value": "00:xx:29:28:xx:c5",
+                    "x_nozomi_info_ref": "7"
+                },
+                "6": {
+                    "type": "x-nozomi-info",
+                    "zone": "ProdNet-A",
+                    "roles": "consumer, terminal",
+                    "is_public": false,
+                    "is_node_learned": true,
+                    "is_reputation_bad": false
+                },
+                "7": {
+                    "type": "x-nozomi-info",
+                    "zone": "ProdNet-A",
+                    "roles": "other",
+                    "is_public": false,
+                    "is_node_learned": true,
+                    "is_reputation_bad": false
+                },
+                "8": {
+                    "type": "ipv4-addr",
+                    "value": "1.1.1.1"
+                }
+            },
+            "first_observed": "2024-01-09T09:19:18.000Z",
+            "last_observed": "2024-01-09T09:19:18.000Z",
+            "number_observed": 1
+        },
+		{
+            "id": "observed-data--8e22ced6-7f5c-4253-b211-1eaebf00dafa",
+            "type": "observed-data",
+            "created_by_ref": "identity--f431f809-377b-45e0-aa1c-6a4751cae5ff",
+            "created": "2024-02-22T06:51:20.244Z",
+            "modified": "2024-02-22T06:51:20.244Z",
+            "objects": {
+                "0": {
+                    "type": "x-ibm-finding",
+                    "alert_id": "d2831312-17a9-53f2-b394-059e1de2202f",
+                    "x_is_acknowledged": false,
+                    "severity": 50,
+                    "time_observed": "2024-01-09T09:19:33.000Z",
+                    "dst_ip_ref": "1",
+                    "src_ip_ref": "3",
+                    "x_alert_status": "open",
+                    "name": "New global MAC vendor",
+                    "ioc_refs": [
+                        "8"
+                    ],
+                    "ttp_tagging_refs": [
+                        "9"
+                    ],
+                    "description": "A new Private MAC Address has been found in the network -- 52:xx:00:12:xx:02",
```

Comment:
Please use a valid but clearly false MAC address, such as "01:01:01:01:01:01"

Reply:
Updated as per the suggestion.